src/DI/ContainerBuilder.php

```diff
@@ -445,6 +445,18 @@ private function resolveEntityClass($entity, $recursive = array())
 	}
 
 
+	private function checkCase($class)
```

Comment:
so, this throws an error, when I supply wrongly cased classname in config or into service definition?